cms/templates/js/settings-tab-section.underscore

```diff
@@ -0,0 +1,10 @@
+<ul class="settings-tab">
+    <li class="settings-section">
+        <a id="general_settings" class="settings-tab-link" href="#"><%- gettext('General Settings') %></a>
+    </li>
+    <li class="settings-section">
+        <a id="advanced_settings" class="settings-tab-link" href="#"><%- gettext('Advanced') %></a>
```

Comment:
I find it confusing that that there is a class called "advanced_settings" which can be applied to a div, but that there is an id with the exact same name which isn't used for the same element. These need to be named differently to make it clear when they should be used. My preference would be that these be called "advanced-settings-button", and I would switch from an id to a class in case we ever need another "Advanced" button. In general, I recommend never using ids as they cause so many problems when there are duplicates.